[2018-10-30 22:51:25] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:51:25] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:51:26] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function setName() on null" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 61 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function setName() on null at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:61)"} []
[2018-10-30 22:52:58] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:52:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:52:58] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function setName() on null" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 61 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function setName() on null at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:61)"} []
[2018-10-30 22:54:13] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:54:13] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:54:13] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected 'public' (T_PUBLIC)" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 109 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected 'public' (T_PUBLIC) at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:109)"} []
[2018-10-30 22:54:16] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:54:16] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:54:16] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected 'public' (T_PUBLIC)" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 109 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected 'public' (T_PUBLIC) at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:109)"} []
[2018-10-30 22:54:52] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:54:52] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:54:52] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected 'public' (T_PUBLIC)" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 109 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected 'public' (T_PUBLIC) at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:109)"} []
[2018-10-30 22:54:56] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:54:56] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:54:56] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected 'public' (T_PUBLIC)" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 109 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected 'public' (T_PUBLIC) at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:109)"} []
[2018-10-30 22:55:04] request.INFO: Matched route "inee_get_dimension". {"route":"inee_get_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::getDimensionAction","id":"undefined","_route":"inee_get_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/get-dimension/undefined","method":"GET"} []
[2018-10-30 22:55:04] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:55:04] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected 'public' (T_PUBLIC)" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 109 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected 'public' (T_PUBLIC) at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:109)"} []
[2018-10-30 22:55:25] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:55:25] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:55:25] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected 'public' (T_PUBLIC)" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 109 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected 'public' (T_PUBLIC) at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:109)"} []
[2018-10-30 22:58:57] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:58:57] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:58:57] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected ')'" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 59 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected ')' at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:59)"} []
[2018-10-30 22:59:46] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 22:59:46] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 22:59:46] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected ')'" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 59 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected ')' at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:59)"} []
[2018-10-30 23:02:23] request.INFO: Matched route "inee_create_dimension". {"route":"inee_create_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\IneeController::createDimensionAction","_route":"inee_create_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/inee/create-dimension","method":"POST"} []
[2018-10-30 23:02:23] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-10-30 23:02:23] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function setName() on null" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php line 61 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function setName() on null at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/IneeController.php:61)"} []
[2018-10-31 13:36:28] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-10-31 13:36:28] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-10-31 13:36:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-10-31 13:36:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-10-31 13:37:46] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-10-31 13:37:46] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-11-01 10:27:56] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-11-01 10:27:56] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-08 18:55:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-08 18:55:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-08 23:06:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-08 23:06:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-08 23:07:28] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png"" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 10:17:59] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 10:55:33] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 10:55:33] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 11:38:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 11:38:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 11:40:06] request.INFO: Matched route "questionnaire_inee_dimension". {"route":"questionnaire_inee_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::getDimensionsAction","_route":"questionnaire_inee_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/get-dimensions","method":"GET"} []
[2018-12-12 11:40:06] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 11:40:06] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "rray_push" from namespace "AppBundle\Services"." at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Services/IneeResults.php line 60 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"rray_push\" from namespace \"AppBundle\\Services\". at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Services/IneeResults.php:60)"} []
[2018-12-12 11:40:36] request.INFO: Matched route "questionnaire_inee_dimension". {"route":"questionnaire_inee_dimension","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::getDimensionsAction","_route":"questionnaire_inee_dimension"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/get-dimensions","method":"GET"} []
[2018-12-12 11:40:36] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 11:40:36] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "rray_push" from namespace "AppBundle\Services"." at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Services/IneeResults.php line 60 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"rray_push\" from namespace \"AppBundle\\Services\". at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Services/IneeResults.php:60)"} []
[2018-12-12 11:50:39] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 11:50:39] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 12:26:24] request.INFO: Matched route "questionnaire_inee_export_pdf". {"route":"questionnaire_inee_export_pdf","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::exportPdfAction","teacherId":"35","_route":"questionnaire_inee_export_pdf"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/export-pdf/35","method":"GET"} []
[2018-12-12 12:26:24] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 12:26:24] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Cannot access protected property AppBundle\Services\PDF::$w" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php line 470 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Cannot access protected property AppBundle\\Services\\PDF::$w at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php:470)"} []
[2018-12-12 12:28:15] request.INFO: Matched route "questionnaire_inee_export_pdf". {"route":"questionnaire_inee_export_pdf","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::exportPdfAction","teacherId":"35","_route":"questionnaire_inee_export_pdf"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/export-pdf/35","method":"GET"} []
[2018-12-12 12:28:15] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 12:28:16] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "MultiCellBltArray" of class "AppBundle\Services\PDF"." at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php line 485 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"MultiCellBltArray\" of class \"AppBundle\\Services\\PDF\". at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php:485)"} []
[2018-12-12 12:29:37] request.INFO: Matched route "questionnaire_inee_export_pdf". {"route":"questionnaire_inee_export_pdf","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::exportPdfAction","teacherId":"35","_route":"questionnaire_inee_export_pdf"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/export-pdf/35","method":"GET"} []
[2018-12-12 12:29:37] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 12:29:38] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "MultiCellBltArray" from namespace "AppBundle\Controller"." at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php line 485 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"MultiCellBltArray\" from namespace \"AppBundle\\Controller\". at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php:485)"} []
[2018-12-12 12:33:40] request.INFO: Matched route "questionnaire_inee_export_pdf". {"route":"questionnaire_inee_export_pdf","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::exportPdfAction","teacherId":"35","_route":"questionnaire_inee_export_pdf"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/export-pdf/35","method":"GET"} []
[2018-12-12 12:33:40] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 12:33:40] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "MultiCellBltArray" of class "AppBundle\Services\PDF"." at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php line 487 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"MultiCellBltArray\" of class \"AppBundle\\Services\\PDF\". at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php:487)"} []
[2018-12-12 12:34:58] request.INFO: Matched route "questionnaire_inee_export_pdf". {"route":"questionnaire_inee_export_pdf","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::exportPdfAction","teacherId":"35","_route":"questionnaire_inee_export_pdf"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/export-pdf/35","method":"GET"} []
[2018-12-12 12:34:58] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 12:34:59] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Cannot access protected property AppBundle\Services\PDF::$x" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php line 487 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Cannot access protected property AppBundle\\Services\\PDF::$x at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php:487)"} []
[2018-12-12 12:37:33] request.INFO: Matched route "questionnaire_inee_export_pdf". {"route":"questionnaire_inee_export_pdf","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionnaireIneeController::exportPdfAction","teacherId":"35","_route":"questionnaire_inee_export_pdf"},"request_uri":"https://apitest.diagnosticosxxi.org/questionnaire-inee/export-pdf/35","method":"GET"} []
[2018-12-12 12:37:33] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-12 12:37:34] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Cannot access protected property AppBundle\Services\PDF::$x" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php line 487 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Cannot access protected property AppBundle\\Services\\PDF::$x at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionnaireIneeController.php:487)"} []
[2018-12-12 13:12:29] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 14:59:29] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 14:59:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 15:01:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 15:06:20] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 15:07:48] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 18:42:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 18:42:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 18:54:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 18:54:55] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 20:14:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-12 20:14:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-13 08:58:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-13 08:58:31] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-13 08:58:55] request.INFO: Matched route "questionary_century_list_by_area". {"route":"questionary_century_list_by_area","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionCenturyController::getQuestionByAreaAction","areaId":"11","userId":"38","_route":"questionary_century_list_by_area"},"request_uri":"https://apitest.diagnosticosxxi.org/questionary-century/list-by-area/11/38","method":"GET"} []
[2018-12-13 08:58:55] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-13 08:58:55] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function getId() on null" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionCenturyController.php line 129 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function getId() on null at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionCenturyController.php:129)"} []
[2018-12-13 09:00:51] request.INFO: Matched route "questionary_century_list_by_area". {"route":"questionary_century_list_by_area","route_parameters":{"_controller":"AppBundle\\Controller\\QuestionCenturyController::getQuestionByAreaAction","areaId":"11","userId":"38","_route":"questionary_century_list_by_area"},"request_uri":"https://apitest.diagnosticosxxi.org/questionary-century/list-by-area/11/38","method":"GET"} []
[2018-12-13 09:00:51] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-12-13 09:00:51] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function getId() on null" at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionCenturyController.php line 129 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function getId() on null at /home/dsxxi/public_html/desarrollo/api/src/AppBundle/Controller/QuestionCenturyController.php:129)"} []
[2018-12-13 09:01:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/1536262995.png" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/1536262995.png\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
[2018-12-13 09:01:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /uploads/courses/" (from "https://test.diagnosticosxxi.org/suggested-course")" at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php line 2817 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /uploads/courses/\" (from \"https://test.diagnosticosxxi.org/suggested-course\") at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/classes.php:2817, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /home/dsxxi/public_html/desarrollo/api/var/cache/prod/appProdProjectContainerUrlMatcher.php:1314)"} []
